cost_calculator/app.py

```diff
@@ -15,6 +15,8 @@
 WEEK = 'week'
 MONTH = 'month'
 YEAR = 'year'
+KIP_NODE_LABEL_KEY = 'app'
+KIP_NODE_LABEL_VALUE = 'kip-provider'
```

Comment:
This is not the best label, we should instead use `type: virtual-kubelet` to find any virtual node in the cluster.